apps/crn-server/src/data-providers/users.data-provider.ts

```diff
@@ -492,6 +502,11 @@ export const parseGraphQLUserToDataObject = (
         : undefined,
     createdDate,
     orcid,
+    dismissedGettingStarted:
```

Comment:
wouldnt this be enough?
```suggestion
    dismissedGettingStarted: !!item.flatData?.dismissedGettingStarted
```

Reply:
Committed.